### PR TITLE
MInor edits to Magento 2.3.4 Release Notes

### PR DESCRIPTION
**Pull request by jfrontain**
This PR adds a new highlights section for PWA Studio, and additional highlight for dotdigital, and additional links into the devdocs doc set.

### CODE REVIEW

**Comment by jcalcaben** [comment]
other than the PageBuilder stuff, the PWA Studio changes apply to both editions

File: src/guides/v2.3/release-notes/release-notes-2-3-4-commerce.md [hunk 1]
```diff
@@ -116,6 +116,22 @@ See [Release notes](https://devdocs.magento.com/guides/v2.3/graphql/release-note
...
 
+PWA and Page Builder integration lets you use Page Builder to define your layout and render that content in a PWA Studio storefront
+
+*  Improved getting-started experience through the use of `@magento/create-pwa` to scaffold your initial project using Venia as your template
+
+*  Separation of the logic (Talons) and presentation pieces (venia-ui) of certain React hooks in Peregrine. Developers can now swap out either the logic or the presentation side of a component
+
+*  Routing now handled through the React Router (library of navigational components)
+
+*  Refactored Venia [state management}(https://magento.github.io/pwa-studio/technologies/basic-concepts/state-management/) to abstract and reduce dependency on Redux
```
[the remark this answers is not in the document]
Comment (jfrontain) [reply src/guides/v2.3/release-notes/release-notes-2-3-4-commerce.md]: thanks

**Comment by jfrontain** [comment]
thanks. I had approval from Andrew to add this text to the Commerce notes, but will add the relevant sections to the OS notes, too.

**Comment by jfrontain** [comment]
running tests

**Pull request merged**